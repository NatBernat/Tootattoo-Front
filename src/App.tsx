const App = () => {
  return (
    <div className="App">
      <h1>Tootattoo</h1>
      <p>Site under contruction</p>
    </div>
  );
};

export default App;
