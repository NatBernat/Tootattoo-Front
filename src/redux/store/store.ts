import { configureStore } from "@reduxjs/toolkit";
import usersReducer from "../features/userSlice/userSlice";
import tattoosReducer from "../features/tattoosSlice/tattoosSlice";

const store = configureStore({
  reducer: {
    user: usersReducer,
    tattoos: tattoosReducer,
  },
});

export type RootState = ReturnType<typeof store.getState>;
export type AppDispatch = typeof store.dispatch;

export default store;
