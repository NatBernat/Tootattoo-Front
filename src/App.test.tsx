import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import { Provider } from "react-redux";
import App from "./App";
import store from "./redux/store/store";

describe("Given App component", () => {
  describe("When it's instantiated", () => {
    test("Then it should render a header", () => {
      render(
        <Provider store={store}>
          <App />
        </Provider>
      );
      const testedHeader = screen.getByRole("heading");

      expect(testedHeader).toBeInTheDocument();
    });
  });
});
