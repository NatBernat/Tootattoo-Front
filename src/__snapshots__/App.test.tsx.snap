// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a App component page When it's instantiated Then it should always match this snapshot 1`] = `
<div
  className="App"
>
  <h1>
    Tootattoo
  </h1>
  <p>
    Site under contruction
  </p>
</div>
`;
